### PR TITLE
chore(netflix): convert override module to TS

### PR DESCRIPTION
**Pull request by icfantv**
* convert template override module to typescript.

### CODE REVIEW

**Comment by icfantv** [comment]
@anotherchrisberry @jrsquared PTAL.

Also alphabetized the imports/module names in the netflix module.

File: app/scripts/modules/netflix/netflix.module.ts [hunk 1]
```diff
@@ -1,18 +1,16 @@
 import {module} from 'angular';
 
-import {
-  APPLICATION_DATA_SOURCE_REGISTRY,
-  ApplicationDataSourceRegistry
-} from 'core/application/service/applicationDataSource.registry';
-import {CLOUD_PROVIDER_REGISTRY, CloudProviderRegistry} from 'core/cloudProvider/cloudProvider.registry';
-import {NETFLIX_APPLICATION_MODULE} from './application';
+import {NetflixSettings} from './netflix.settings';
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/netflix.module.ts]: Not quite alphabetized but close enough

**Comment by anotherchrisberry** [comment]
LGTM

File: app/scripts/modules/netflix/netflix.module.ts (hunk 1 above)
Comment (icfantv) [inline app/scripts/modules/netflix/netflix.module.ts]: My daughter was helping me.  :-)

**Pull request merged**